datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestStickyMulticastStrategy.java

```diff
@@ -271,7 +272,7 @@ public void testDontCreateNewTasksWhenCalledSecondTime() {
   public void testDontCreateNewTasksWhenInstanceChanged() {
     String[] instances = new String[]{"instance1", "instance2", "instance3"};
     List<DatastreamGroup> datastreams = generateDatastreams("ds", 5);
-    StickyMulticastStrategy strategy = new StickyMulticastStrategy(Optional.empty(), Optional.empty());
+    StickyMulticastStrategy strategy = new StickyMulticastStrategy(Optional.empty());
```

Comment:
e.g. of not being consistent with the direct use of "new StickyMulticastStrategy" vs. the function "createStickyMulticastStrategyObject"